extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1425,7 +1425,7 @@ private static function isRegexp ($needle) {
         return false;
     }
 
-    public static function dplReplaceParserFunction(&$parser, $text, $pat, $repl='') {
+    public static function dplReplaceParserFunction($parser, $text, $pat, $repl='') {
 		if ($text=='' || $pat=='') return '';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)